Extension/src/LanguageServer/extension.ts

```diff
@@ -1062,6 +1110,37 @@ function handleMacCrashFileRead(err: NodeJS.ErrnoException | undefined | null, d
     logMacCrashTelemetry(data);
 }
 
+async function handleCrashFileRead(crashDirectory: string, crashFile: string, err: NodeJS.ErrnoException | undefined | null, data: string): Promise<void> {
+    if (err) {
+        if (err.code === "ENOENT") {
+            return; // ignore known issue
+        }
+        return logCppCrashTelemetry("readFile: " + err.code);
+    }
+
+    const lines: string[] = data.split("\n");
+    data = crashFile + "\n";
+    const filtPath: string | null = which.sync("c++filt", { nothrow: true });
+    for (let line of lines)
+    {
+        line = line.trim();
+        if (filtPath && line.startsWith("_")) {
+            line = (await util.spawnChildProcess(filtPath, [line])).output;
```

Comment:
If `spawnChildProcess` can fail, we should make sure that we `.catch()` the error and exit gracefully.

Reply:
done